### ISSUE TITLE
Query editing helper freezes when quotation marks are added or removed

### ISSUE THREAD

**Issue by dan-zeman** [ufal/perl-pmltq-web#40]
Moving this from #38 to a separate thread, as it is a separate issue:

The page hung up when I tried to edit the query and pressed the backspace key (I think there was something like form="ten" and I was just about to erase the final quotation mark).

In case it matters: I am using Chrome Verze 44.0.2403.157 m on Windows 7 64bit. And the error occurred again. It seems that the problem is related to quotation marks and it occurs if there is already a query, I decide to edit it, during editing the number of quotation marks temporarily becomes uneven and the JavaScript editing helper (which is quite useful otherwise!) tries to solve it but only manages to freeze the page.

**Comment by m1ch4ls** [comment]
This happens when filter and query is on the same line..
```
a-node $a := [lemma = "všechen"] >> for lower($a.form), $a.tag give $1, $2, count() sort by $1
```

**Comment by matyaskopp** [comment]
I have probably fixed the bug: https://github.com/ufal/perl-pmltq-web/commit/45ded406453734664f2105c3a9fa6fd79f470bef#diff-39fb0f4d21c1b1d32cf077331208b668 . There were wrong regexp evaluation. I thing that it shouldn't be necessarily infinity loop but the evaluation takes a lot of time.

**Comment by m1ch4ls** [comment]
Confirming it's fixed by 45ded406453734664f2105c3a9fa6fd79f470bef
